CHANGELOG.md

```diff
@@ -7,16 +7,37 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 This changelog also contains important changes in dependencies.
 
 ## [Unreleased]
+This is the first release under the stewardship of [Linebender][], who is now responsible for maintenance of this crate.
```

Comment:
Should this link have a corresponding definition?

Same for the other links in this change

Reply:
Nevermind apparently they are all at the bottom. Then I don't get the second pair of brackets. It seems unnecessary.